packages/web/src/components/host-remix-contest-modal/HostRemixContestModal.tsx

```diff
@@ -63,11 +82,21 @@ export const HostRemixContestModal = () => {
   const hasContestEntries = remixesLoading || remixes?.length
   const displayTurnOffButton = !hasContestEntries && isEdit
 
+  const remixContestData = remixContest?.eventData as RemixContestData
```

Comment:
how come this isn't typed?

Reply:
there is no specific type of remix contests, only events so the eventData can look like anything. I do want to add a common spot to hold the type though

Reply:
yeah i think if we can plop that type into the `useRemixContest` that would be ideal. then long term we can think about giving back types from the backend